### PR TITLE
add avatarUrl in the peoplePreviewList response

### LINKED ISSUE #2316: Add `avatarUrl` in the `peoplePreviewList` response object
To add them to the `RoomCard`

### PR DESCRIPTION
**Pull request by amitojsingh366**
- closes #2316
- adds typings for `userCreateBot` in kebab

### CODE REVIEW

**Comment by ajmnz** [comment]
@dark-elixir-shibe tests are failing, letting you know as per your comment in #2280

**Comment by amitojsingh366** [comment]
> 
> 
> @dark-elixir-shibe tests are failing, letting you know as per your comment in #2280

i feel someone has broken tests again...

**Comment by goldyydev** [comment]
> > @dark-elixir-shibe tests are failing, letting you know as per your comment in #2280
> 
> i feel someone has broken tests again...

It has been like that for a while, take a look at merged elixir PRs from the past few days

**Comment by benawad** [comment]
pull staging and tests should pass

**Comment by amitojsingh366** [comment]
> pull staging and tests should pass

ok, lemme try

**Comment by benawad** [comment]
try once more

**Comment by amitojsingh366** [comment]
> try once more

ok...

File: kebab/src/entities.ts [hunk 1]
```diff
@@ -8,9 +8,10 @@ export type RoomPeer = {
 };
 
 export type UserPreview = {
-  numFollowers: number;
   id: UUID;
   displayName: string;
+  numFollowers: number;
+  avatarUrl: string
```
Comment (benawad) [inline kebab/src/entities.ts]: this need to be `string | null`

**Review by benawad — CHANGES_REQUESTED** [review]

File: kebab/src/entities.ts (hunk 1 above)
Comment (amitojsingh366) [reply kebab/src/entities.ts]: done

**Pull request merged**